tests/core/editor/manual/detaching.html

```diff
@@ -0,0 +1,186 @@
+<style>
+	form ul {
+		list-style: none;
+	}
+	input[disabled]+label {
+		text-decoration: line-through;
+	}
+</style>
+
+<form id="controls">
+	<div id="type">
+		<h2>Editor type:</h2>
+		<ul>
+			<li><input type="radio" name="editor" value="classic" checked><label>Classic</label></li>
+			<li><input type="radio" name="editor" value="inline"><label>Inline </label></li>
+			<li><input type="radio" name="editor" value="divarea"><label>Divarea</label></li>
+		</ul>
+	</div>
+	<div id="element">
+		<h2>Editor element:</h2>
+		<ul>
+			<li><input type="radio" name="element" value="textarea" checked><label>Textarea</label></li>
+			<li><input type="radio" name="element" value="div"><label>Div</label></li>
+		</ul>
+	</div>
+	<div id="destroy">
+		<h2>Destroy:</h2>
+		<ul>
+			<li><input type="radio" name="destroy" value="0" checked><label>synchronously</label></li>
+			<li><input type="radio" name="destroy" value="1"><label>async with 0 timeout</label></li>
+			<li><input type="radio" name="destroy" value="2"><label>before 'loaded' event</label></li>
+			<li><input type="radio" name="destroy" value="3"><label>before 'scriptLoader.load' callback called</label></li>
+			<li><input type="radio" name="destroy" value="4"><label>before iframe#onload</label></li>
+		</ul>
+	</div>
+</form>
+<button id="createdestroy">Create editor</button>
```

Comment:
As you already created `form` for controlling radio buttons, shouldn't this button be a part of form also?

Reply:
Button inside form has default action that refreshes the page. By having it outside of form we don't need to `evt.preventDefault()`.